README.md

```diff
@@ -188,113 +270,104 @@ The following CSS:
 }
 ```
 
-will be processed to:
+The above will be transformed into this:
 
 ```css
 .box-foo {
 	width: 150px;
+}
 
-	.box-bar {
-		width: 150px;
-	}
-
-	@media (max-width: 800px) {
-		.box-bar {
-			width: 300px;
-		}
-	}
+.box-foo .box-bar {
+	width: 150px;
 }
 
 @media (max-width: 800px) {
 	.box-foo {
 		width: 300px;
 	}
+	
+	.box-foo .box-bar {
+		width: 300px;
+	}
 }
 ```
 
-
-
-## Interoperability
-
-`postcss-css-variables` plays really nice with [`postcss-nested`](https://github.com/postcss/postcss-nested) in order to get a larger subset of CSS variables features. *See [Nested rules, Usage section](#nested-rules)*
-
-If you are using [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties) previously, we have a compatible feature set and more so you can switch over without having to refactor any of your code. You can just start writing the new awesome stuff.
-
-
 # Why?
 
 This plugin was spawned out of a [discussion on the `cssnext` repo](https://github.com/cssnext/cssnext/issues/99) and a personal need.
 
-There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), which at the moment, covers less of the [CSS-variable/custom-property draft/spec](http://dev.w3.org/csswg/css-variables/) compared to this plugin. It also happened to not cover the feature I was after and after dealing with it, making an incomplete prototype piggybacking off of it; I decided to write my own starting from scratch. I do use some of the same/similar unit tests because that just makes sense to get proper code coverage.
+There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), although it restricts itself much more than this, preferring spec conformance over imperfect feature support.
```

Comment:
> There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), although it restricts itself much more than this plugin, preferring partial spec conformance. This plugin has the same capabilities but also adds imperfect feature support which stem from not being to know what the DOM will look like when you compile your CSS. We instead look at the explicit structure of your CSS selectors.

It also doesn't seem to read very well IMO. It gives at least me the impression almost of "this is superior because the other is crap". That's the other reason I'm hesitant to add that in. It just comes off as a little implicitly condescending over the other plugin, to be honest.